src/pages/news-conteiner/components/news-list/news-page-item.style.js

```diff
@@ -0,0 +1,56 @@
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
There is no need to put `page` in nested components. Please, rename files to format `news-item`

Reply:
done